profiles/migrations/0008_add_first_name_and_last_name_to_profile.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.3 on 2019-10-24 11:26
+
+from django.db import migrations, models
+
+
+def populate_name_from_user_for_existing_profiles(apps, schema_editor):
+    Profile = apps.get_model("profiles", "Profile")
+    for profile in Profile.objects.all():
+        if profile.user is not None:
+            profile.first_name = profile.user.first_name or profile.first_name
+            profile.last_name = profile.user.last_name or profile.last_name
+        profile.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("profiles", "0007_add_legal_relationship_model")]
+
+    operations = [
+        migrations.AddField(
+            model_name="profile",
+            name="first_name",
+            field=models.CharField(blank=True, max_length=255, null=True),
+        ),
+        migrations.AddField(
+            model_name="profile",
+            name="last_name",
+            field=models.CharField(blank=True, max_length=255, null=True),
+        ),
+        migrations.RunPython(populate_name_from_user_for_existing_profiles),
```

Comment:
I think you should explicitly specify `migrations.RunPython.noop` as the reverse operation here, or django won't allow running migrations backwards. or do I misremember? 🤔